opdracht-1.js

```diff
@@ -0,0 +1,22 @@
+// Array met tv-type namen en uitkomst in console loggen.
+const tvTypes = inventory.map((tvType) => tvType.type);
+
+console.log(tvTypes);
+
+// Array met alle uitverkochte objecten en uitkomst in de console loggen.
+
+const soldOutTelevisions = inventory.filter((soldOutTelevision) => soldOutTelevision.sold === soldOutTelevision.originalStock);
+
+console.log(soldOutTelevisions);
+
+// Array met alle TV objecten die AmbiLight hebben en uitkomst in de console loggen.
+
+const ambiLightTelevisions = inventory.filter((ambiLightTelevision) => ambiLightTelevision.options.ambiLight === true);
```

Comment:
Deze kan nog iets korter
```suggestion
const ambiLightTelevisions = inventory.filter(ambiLightTelevision => ambiLightTelevision.options.ambiLight);
```